### ISSUE TITLE
Forceclose I think? 

### ISSUE THREAD

**Issue by Cashstachestashcache** [MutinyWallet/mutiny-web#1199]
I hear 3 channels and I just opened the app to check my recieves from the fedimint

And my balance is greatly reduced! I didn't go into the setting recently to forceclose anything myself. And I've checked back daily so I don't THINK i had any reason for a forceclose... 

I clicked on my profile and I do see the amount from the big channels as being on chain *phew*
But I can't see how to access it from there even at the moment. Maybe because it just happened? But on the main page my amount being so reduced really freaked me out for a second. 

I see another forceclose post where you're asking for logs so I'll include them below. 
[mutiny-logs (1).txt](https://github.com/user-attachments/files/15745958/mutiny-logs.1.txt)

**Comment by benthecarman** [comment]
The logs you provider are only for 6/1 and 6/2, they don't go up until today, can you try re-uploading them?

Also if you click on the force close it should tell you the reason.

The balance is not spendable immediately but should be available within 3-14 days

**Comment by Cashstachestashcache** [comment]
I don't see how to click the foreclose? 

It doesn't show in my history of that's where it's supposed to show up. 

I'll redownload the logs now.

**Comment by Cashstachestashcache** [comment]
![Screenshot_20240607_174709_Mutiny Wallet](https://github.com/MutinyWallet/mutiny-web/assets/169390930/22554f72-880e-483b-bb5d-f2be2dd8c248)
[mutiny-logs (2).txt](https://github.com/user-attachments/files/15746006/mutiny-logs.2.txt)

**Comment by benthecarman** [comment]
Hmm these logs go up to now but I don't see any logs of the force closure actually happening. Are you using mutiny on multiple devices, it's possible the event happened on there

**Comment by Cashstachestashcache** [comment]
No sir

I only use it on my phone.

But this does happen from time to time I've notice also that I might get a zap notification on Nostr then go check my wallet and it will have the most recent one but maybe not the left could hours or so. Usually it updates though after a little bit. But the next time I look out a color hours later.

**Comment by Cashstachestashcache** [comment]
In fact looking now (screenshot above) I received a 696 zap from walker today that isn't showing up in that list of transactions.

**Comment by benthecarman** [comment]
Hmmm I haven't seen that before. Do you at least have an on-chain balance on your profile screen? Worst case it'll sweep back the funds in 3-14 days

**Comment by Cashstachestashcache** [comment]
Yes! Thank God! 

And thank you for looking into it. 

But yes. It does say least show that it's on chain on my profile screen. What have me nervous about the onchain part was not being able to swipe it to the side (like I was able to in the past) as if I was going to send it. 

But I saw in the other issue topic that you had said it's not spendable for a few days. So I was like ok "phews* that is at least good.
